src/app/app.component.html

```diff
@@ -1,33 +1,33 @@
 <div class="example-container" [class.example-is-mobile]="mobileQuery.matches">
 
   <mat-toolbar color="primary" class="example-toolbar">
-    <span>Dakar</span>
+    <span>Dakar </span>
     <button mat-icon-button color="warn" (click)="snav.toggle()">
       <mat-icon aria-label="Example icon-button with a menu icon">menu</mat-icon>
     </button>
-    <h1 class="example-app-name">Responsive App</h1>
+    <h1 class="example-app-name">{{title}}</h1>
   </mat-toolbar>
 
   <mat-sidenav-container class="example-sidenav-container"
                          [style.marginTop.px]="mobileQuery.matches ? 56 : 0">
 
-    <mat-sidenav #snav [mode]="mobileQuery.matches ? 'over' : 'side'"
+    <mat-sidenav opened #snav [mode]="mobileQuery.matches ? 'over' : 'side'"
```

Comment:
what is "opened" ?

Reply:
it's an attribute of mat-sidenav proposed by Angular Material